src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -738,7 +738,7 @@ protected function checkCategories($hookData, Github $github, Logger $logger, $p
 	 */
 	protected function checkFilesAndAssignCategory($files)
 	{
-		$addCategories = [];
+		$addCategories = array();
```

Comment:
Any particular reason for this change?
Array declaration should always be >= PHP 5.4 - we might include a code sniff to ensure this... :wink: